pkg/features/features.go

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package features
+
+import (
+	"context"
+	"log"
+	"time"
+
+	ofinprocess "github.com/open-feature/go-sdk-contrib/providers/go-feature-flag-in-process/pkg"
+	"github.com/open-feature/go-sdk/openfeature"
+	goffclient "github.com/thomaspoignant/go-feature-flag"
+	"github.com/thomaspoignant/go-feature-flag/retriever/k8sretriever"
+	"k8s.io/client-go/rest"
+
+	"github.com/cloudoperators/greenhouse/pkg/clientutil"
+)
+
+func GetOFClient(appName string) *openfeature.Client {
```

Comment:
It would be helpful to have some context to what this client does. Maybe something like this?
```suggestion
// NewClient returns a open-feature client loading feature flags from a ConfigMap
func NewClient(appName string) *openfeature.Client {
```

Reply:
sure, we are not finished fully with this PR, will do some meaningful commenting into the new functions

Reply:
added some comments to the code and renamed the func to better correlate in [4baeee0](https://github.com/cloudoperators/greenhouse/pull/839/commits/4baeee0906ce60627a45e9e085b92cbd28c517fa)